docs/concepts/Alexa-Hosted-Skill-Commands.md

```diff
@@ -0,0 +1,118 @@
+# CREATE SKILL - NEW COMMAND
+
+`askx new` allows developers to create an `Alexa hosted skill`, and clone the skill project to the local machine, which replaces and optimizes `ask create-hosted-skill` command in CLI V1.
+
+Alexa hosted deployer builds Alexa skills quickly by provisioning required resources automatically. 
+Alexa hosted service will create a new AWS account with a deployment pipeline (CodeBuild), a skill Lambda function, a Git repository (CodeCommit) and an S3 bucket for developers.
```

Comment:
Do we need some much details about behind the scenes process?

Reply:
I removed duplicate description, but keep the resource details because these resources are good to know for users.